### ISSUE TITLE
Unable to install cert-manager using pyhelm

### ISSUE THREAD

**Issue by cyr62110** [flaper87/pyhelm#44]
Hi,

I am trying to install cert-manager using PyHelm
```
> helm search cert-manager
NAME               	CHART VERSION	APP VERSION	DESCRIPTION                  
stable/cert-manager	v0.5.2       	v0.5.2     	A Helm chart for cert-manager
```

When I try to instantiate the ChartBuilder with the chart information like following:
```python
from pyhelm import chartbuilder
chart = { "name": "cert-manager", "version": "v0.5.2", "source": { "type": "repo", "location": "https://kubernetes-charts.storage.googleapis.com" }}
chartb = chartbuilder.ChartBuilder(chart)
```

I get the following exception:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/site-packages/pyhelm/chartbuilder.py", line 47, in __init__
    self.source_directory = self.source_clone()
  File "/usr/local/lib/python2.7/site-packages/pyhelm/chartbuilder.py", line 90, in source_clone
    self.chart.source.headers)
  File "/usr/local/lib/python2.7/site-packages/pyhelm/repo.py", line 131, in from_repo
    metadata = sorted(versions, key=_semver_sorter)[-1]
  File "/usr/local/lib/python2.7/site-packages/pyhelm/repo.py", line 37, in _semver_sorter
    return list(map(int, x['version'].split('.')))
ValueError: invalid literal for int() with base 10: 'v0'
```

The installation using helm command line is working properly.
```
> helm install stable/cert-manager --version v0.5.2 --set rbac.create=true
NAME:   illmannered-kudu
LAST DEPLOYED: Mon Jan 14 08:07:58 2019
NAMESPACE: default
STATUS: DEPLOYED
```

Best,

**Comment by ljakimczuk** [comment]
Hi @cyr62110 , it seems that's due to the 'v' tag in the version, not all charts on the stable repository have this tag, so maybe it is used for some special purpose. Anyway, it does not contribute to the version, so try to use the bare "0.5.2" version.

**Comment by cyr62110** [comment]
Hi,

Unfortunately using bare "0.5.2" is not working. I have the following trace:
```
2019-01-14 23:46:29,994 - ChartBuilder - INFO - Cloning https://kubernetes-charts.storage.googleapis.com/ for release cert-manager
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/site-packages/pyhelm/chartbuilder.py", line 47, in __init__
    self.source_directory = self.source_clone()
  File "/usr/local/lib/python2.7/site-packages/pyhelm/chartbuilder.py", line 90, in source_clone
    self.chart.source.headers)
  File "/usr/local/lib/python2.7/site-packages/pyhelm/repo.py", line 147, in from_repo
    raise VersionError(version)
pyhelm.repo.VersionError: Chart version 0.5.2 not found
```

Best,

**Comment by yanivoliver** [comment]
This should've been resolved by #53. Could you give it a try?

**Comment by yanivoliver** [comment]
Closing due to inactivity.
